src/main/java/com/example/dto/CategoryDto.java

```diff
@@ -0,0 +1,9 @@
+package com.example.dto;
+
+import lombok.Data;
+
+@Data
+public class CategoryDto {
+    private String name;
```

Comment:
id missed. Use separate dto for response and request (add validators to request)